Fhi.HelseId.Tests/HelseIdWebKonfigurasjonTests.cs

```diff
@@ -20,7 +20,7 @@ public void HelseIdWebConfigurationTest()
                 Assert.That(sut.AuthUse, Is.True, "2");
                 Assert.That(sut.UseHttps, Is.True, "3");
                 Assert.That(sut.UseHprNumber, Is.False, "4");
-                Assert.That(sut.AllScopes, Has.Count.GreaterThan(7), "5");
+                Assert.That(sut.AllScopes, Has.Count.GreaterThan(6), "5");
```

Comment:
Should we also add a test when UseHprNumber is true, that this is set to 7?

Reply:
Agree, will do